resources/assets/less/bem/page-extra-tabs.less

```diff
@@ -4,7 +4,7 @@
 .page-extra-tabs {
   position: sticky;
   top: @nav2-height--pinned;
-  z-index: 1;
+  z-index: 999;
```

Comment:
z-index should be listed in `variables.less` instead so it's easier to check the order site-wide

Reply:
Moved and change it to 99. Seems working 🤔